samples/client/petstore/scala/sttp4/src/main/scala/org/openapitools/client/api/PetApi.scala

```diff
@@ -0,0 +1,178 @@
+/**
+ * OpenAPI Petstore
+ * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+package org.openapitools.client.api
+
+import org.openapitools.client.model.ApiResponse
+import java.io.File
+import org.openapitools.client.model.Pet
+import org.openapitools.client.core.JsonSupport._
+import sttp.client4._
+import sttp.model.Method
+
+object PetApi {
+
+def apply(baseUrl: String = "http://petstore.swagger.io/v2") = new PetApi(baseUrl)
```

Comment:
minor. formatting issue.

Reply:
will try to fix the formatting issue after merging this PR.

Reply:
fixed via https://github.com/OpenAPITools/openapi-generator/pull/15931